src/api/requests.ts

```diff
@@ -120,6 +121,11 @@ async function handleResponse<T>(response: Response): Promise<T> {
     return {} as Promise<T>;
   }
 
+  if (response.headers.get('content-type') === 'application/pdf') {
+    /* TODO: Separate 'get file' returning Blob */
+    return response.blob() as Promise<any>;
```

Comment:
looks like Blob cannot be typed using generics. Maybe add a separate function for file transfer or assume that the user expects Blob when `content-type` is pdf. What do you think @seblj ?

Reply:
Hmm🤔 maybe a separate function to get proper typing, but interesting that it doesn't work as wanted with generics... couldn't really find any information why either by some quick googling